docs-src/concepts/what-is-a-schedule.md

```diff
@@ -28,6 +28,12 @@ Schedules provide a more flexible and user-friendly approach than [Temporal Cron
 A Schedule has an identity and is independent of a Workflow Execution.
 This differs from a Temporal Cron Job, which relies on a cron schedule as a property of the Workflow Execution.
 
+:::caution
+
+The Temporal server enriches responses from scheduled Workflow Executions with additional contextual data, enabling their identification and handoff to subsequent executions. However, if the returned result exceeds the designated block size by temporal overhead (e.g., 2MB - 1KB), it becomes unavailable for use in the subsequent run.
```

Comment:
```suggestion
The Temporal Service enriches responses from scheduled Workflow Executions with additional contextual data, enabling their identification and handoff to subsequent executions.
However, if the returned result exceeds the designated block size by Temporal overhead (e.g., 2MB - 1KB), it becomes unavailable for use in the subsequent run.
```

- We have updated guidance from Product Marketing to use Temporal Service generically as often as possible unless it requires a specific callout to the server.
- Capitalize core terms
- I don't know what "Temporal overhead" means here. 
- Each sentence on a newline.

If my understanding is correct, that my recommendation to provide a simple and approachable message would be more like this:
> If the result returned by the scheduled Workflow Execution exceeds max block size (2MB - 1KB) this result will not be available for the next run.
The max block size is not affected by the extra meta data that Temporal needs to identify the result and provide it to the next Workflow Execution.

Reply:
First, this is an unlikely edge case and does not deserve to be in this high position on the page. It should be down around line 203 (pre-change), at the end of the `### Last completion result` section.

As for the text, I don't really like either of those :slightly_smiling_face:   I'd write

```suggestion
A scheduled workflow may complete with a result up to the maximum blob size (2 MiB by default). However, due to internal limitations, results that are within 1 KiB of this limit cannot be passed to the next execution. So, for example, a workflow that returns a result of size 2,097,000 bytes will be allowed to compete successfully, but that value will not be available as a last completion result. This limitation may be lifted in the future.
```

(capitalize as appropriate)

Reply:
Last, the thing about last completion and AllowAll is going to be wrong soon and should be removed, that's lines 176-190 pre-change.